### PR TITLE
add cuda-version to host and run reqs

### PR DESCRIPTION
**Pull request by gschramm**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by gschramm** [comment]
@conda-forge-admin, please rerender

**Comment by gschramm** [comment]
@jakirkham Following your new issue https://github.com/conda-forge/stir-feedstock/issues/114 on the stir feedstock from yesterday, I create this PR for the parallelproj package (that stir is using). I added the `cuda-version` dependency in `host` & `run` and also a selector `ignore_run_exports: - cudatoolkit`. Can you check whether this is correct?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ outputs:
         - libparallelproj =*=cuda*  # [cuda_compiler_version != "None"]
         - libparallelproj =*=cpu*  # [cuda_compiler_version == "None"]
       ignore_run_exports:
-        - cudatoolkit
+        - cudatoolkit  # [cuda_compiler_version == "11.*"]
```
Comment (jakirkham) [inline recipe/meta.yaml:30]:
```suggestion
        - cudatoolkit  # [(cuda_compiler_version or "None").startswith("11")]
```

**Review by jakirkham — COMMENTED** [review]
Thanks Georg! 🙏

Apologies for the much delayed reply. Was afk for an extended period of time

Generally this looks good

Have included a minor suggestion with the selector below. Maybe this helps with any CI issues encountered previously

File: recipe/meta.yaml (hunk 1 above)
Comment (gschramm) [reply recipe/meta.yaml:30]: many thanks. your help is much appreciated. added your suggestion in a this new PR https://github.com/conda-forge/parallelproj-feedstock/pull/77

File: recipe/meta.yaml [hunk 2]
```diff
@@ -43,8 +43,10 @@ outputs:
       host:
         - libgomp      # [linux]
         - llvm-openmp  # [osx]
+        - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
       run:
         - __cuda  # [cuda_compiler_version != "None"]
+        - cuda-version {{ cuda_compiler_version }}  # [(cuda_compiler_version or "None") != "None"]
```
Comment (jakirkham) [inline recipe/meta.yaml:49]: Note: The `cuda-version` changes were included in PR ( https://github.com/conda-forge/parallelproj-feedstock/pull/75 ). Including relevant code snippet below for reference

https://github.com/conda-forge/parallelproj-feedstock/blob/2e4d60cdf4fb0f18f2073dba5f665df6324488c8/recipe/meta.yaml#L44-L50

**Pull request closed without merging**